tests/snapshots/tests.contrib.grpc.test_grpc.test_method_service.json

```diff
@@ -20,6 +20,7 @@
       "grpc.method.service": "Servicer",
       "grpc.status.code": "StatusCode.OK",
       "language": "python",
+      "network.destination.ip": "::",
```

Comment:
This seems like an unexpected value for an IP address field to have. Is it intentional?

Reply:
`::` is IPv6 loopback range, equivalent to `0.0.0.0` for IPv4, and the [test code that generates the trace](https://github.com/DataDog/dd-trace-py/blob/ba7470328a8f7eccff18bc5e2f71e0c485a98642/tests/contrib/grpc/test_grpc.py#L802-L808) does indeed configure it as IPv6.